assets/scss/components/_header.scss

```diff
@@ -88,10 +88,17 @@ Styleguide Header.Transaction
     text-align: left;
     padding: 0;
   }
+
+  p {
+    font-size: 36px;
```

Comment:
Ahhh - I couldn't find "core" and "bold" was obviously the wrong choice. Thanks - I will update this now